cli/create-scss.js

```diff
@@ -34,6 +34,7 @@ var writeScss = function(iconInfos, scssPath) {
 		var iconInfo = iconInfos[i];
 
 		scss.write('@mixin ' + iconInfo.mixin + '() {\n');
+		scss.write('	@include d2l-icon();\n');
```

Comment:
Only concern I have here is the repeated properties for each icon.  How much duplication do you think we'll see?